chunkset_tpl.h

```diff
@@ -5,6 +5,10 @@
 #include "zbuild.h"
 #include <stdlib.h>
 
+#if CHUNK_SIZE == 32 && defined(X86_SSE41) && defined(X86_SSE2)
```

Comment:
Instead of doing this, maybe just set a preprocessor definition in `chunkset_sse41.c`? 
```c
#if CHUNK_SIZE == 32 && defined(X86_SSE41) && defined(X86_SSE2)
#  define CHUNKSET_QUICKSET \
    if (len <= 16) { \
        return chunkmemset_sse41(out, dist, len); \
    }
#else
#  define CHUNKSET_QUICKSET
#endif
```
Then just call
`CHUNKSET_QUICKSET` in `chunkset_tpl.h`. Or something similar?

Reply:
Then you don't really need that extern I would think.

Reply:
I would still need the extern in the chunkset_avx.c file, though.  And every version of chunkset_*.c would then need to have CHUNKSET_QUICKSET defined.